### ISSUE TITLE
Expandable panel - accessibility - div containing expandable content panel missing accessible markup

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#24]
div containing expandable content panel needs:
role="region"
aria-labelledby=....ID of div expandable button is wrapped in

as per example https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html
